README.md

```diff
@@ -64,6 +64,13 @@ composer require contributte/application
         </br>
         <a href="https://github.com/paveljanda">Pavel Janda</a>
       </td>
+      <td align="center">
+        <a href="https://github.com/mabar">
+            <img width="150" height="150" src="https://avatars0.githubusercontent.com/u/20974277?s=400&v=4">
```

Comment:
size=150?

Reply:
Oh, `s` parameter is size... well, it's wrongly in every single package...